pkg/block/metadata/compactormeta.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package metadata
+
+import "github.com/oklog/ulid"
+
+// CompactorMetaFilename is the known json filename to store compactor metadata.
+const CompactorMetaFilename = "compactor-meta.json"
+
+// CompactorMeta stores block id and when block was marked for deletion.
+type CompactorMeta struct {
+	ID           ulid.ULID `json:"id"`
+	DeletionTime int64     `json:"deletion_time"`
```

Comment:
Comment about using Unix timestamp would be nice here.

Reply:
Yes.